src/main/app/forms/actionForm.ts

```diff
@@ -0,0 +1,17 @@
+import { Form } from 'forms/form'
```

Comment:
Wouldn't a better fix to just be use javascript for adding the row?

Reply:
It's not only the row, there is the calculate total as well. Plus would need to add an XHR endpoint for each of multi-row page we have, this felt like a simpler solution.

Reply:
would have to see what it looks like, is it jumpy or not noticeable to the user?
It really seems like a hack we shouldn't be doing.

Reply:
Unless you meant to append html within JavaScript? Still sounds more complex I think.